<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HttpClient</class>
 <widget class="QWidget" name="HttpClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>718</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HttpClient</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>响应</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QTextBrowser" name="textBrowser"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_5">
     <property name="title">
      <string>请求</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="7">
       <widget class="QComboBox" name="comboBox">
        <item>
         <property name="text">
          <string>GET</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>POST</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>PUT</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>PATCH</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>DELETE</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>HEAD</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>OPTIONS</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QSpinBox" name="spinBox_Port">
        <property name="maximum">
         <number>65535</number>
        </property>
        <property name="value">
         <number>80</number>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEdit_Host"/>
      </item>
      <item row="2" column="5">
       <widget class="QLineEdit" name="lineEdit_Path"/>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>端口号</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>接口路径</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>地址</string>
        </property>
       </widget>
      </item>
      <item row="2" column="8">
       <widget class="QPushButton" name="pushButton_SendRequest">
        <property name="text">
         <string>发送</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="9">
       <widget class="QTableView" name="tableView"/>
      </item>
      <item row="1" column="0" colspan="9">
       <widget class="QLineEdit" name="lineEdit_RequestURL">
        <property name="placeholderText">
         <string>127.0.0.1:80/?a=a&amp;b=b&amp;c=c</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="9">
       <widget class="QComboBox" name="comboBox_2"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
